contracts/common/FundsProxy.sol

```diff
@@ -0,0 +1,44 @@
+pragma solidity 0.4.18;
+
+import "./DelegateProxy.sol";
+import "../lib/zeppelin/token/ERC20.sol";
+import "../lib/misc/ERCProxy.sol";
+
+
+contract FundsProxy is DelegateProxy, ERCProxy {
+    address constant public ETH = 0x0;
+
+    event ProxyDeposit(address sender, uint256 value);
+
+    function getDefaultVault() internal returns (address);
+
+    /**
+     * @notice Send funds to default Vault. This contract should never receive funds,
+     *         but in case it does, this function allows to recover them.
+     * @param _token Token balance to be sent to Vault. ETH(0x0) for ether.
+     */
+    function transferToVault(address _token) external {
+        address vault = getDefaultVault();
+        require(isContract(vault));
+
+        if (_token == ETH) {
+            vault.transfer(address(this).balance);
```

Comment:
Maybe we could make this a proper vault transfer so it gets logged. We just need to convert it to `require(vault.call.value(address(this).balance)())`